### DISCUSSION TITLE
I tried to set the  CornerRadius of the window but it didn't work

### DISCUSSION THREAD

**Discussion by ApaulMyLittleAirport** [AvaloniaUI/Avalonia#8377] [Q&A]
Window prop:   
 HasSystemDecorations="False" CornerRadius="10" Background="yellow" BorderThickness="2" BorderBrush="red"

**Comment by maxkatz6** [comment]
It's not supported. You would need to use native API (AppKit on mac or WinAPI on win) to change these properties.
Though I am not sure if it's even possible on mac and linux. 

Alternatively you can make borderless window (set SystemDecorations to none) and that you can put a Border control with custom corner radius. You will lose some features and native look, but that's it.

**Reply by ApaulMyLittleAirport** [reply]
> 它不受支持。您需要使用本机 API（Mac 上的 AppKit 或 win 上的 WinAPI）来更改这些属性。虽然我不确定这在Mac和Linux上是否可行。
> 
> 或者，您可以创建无边框窗口（将“系统装饰”设置为“无”），并且可以放置具有自定义角半径的边框控件。你会失去一些功能和原生外观，但仅此而已。
Thank you very much for your help, I really benefited a lot, but it seems that after I set it up, there is still no red rounded border.
  SystemDecorations="None" CornerRadius="10" Background="yellow" BorderThickness="12" BorderBrush="red"

**Comment by DasLixou** [comment]
When you are using ```SystemDecorations="None"``` then you can just make the window transparent and put a border with roundings in it.

**Reply by ApaulMyLittleAirport** [reply]
> 

Thank you very much for your advice. I seem to have failed the test. I will try again.
